server/services/Authentication/signup.go

```diff
@@ -0,0 +1,104 @@
+package authentication
+
+import (
+	"fmt"
+	"main/server/db"
+	"main/server/model"
+	"main/server/request"
+	"main/server/response"
+	"main/server/services/token"
+	"main/server/utils"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func AlreadyExists(data string) bool {
```

Comment:
Please add comments as an explanation of what the method does

Reply:
Done